src/DependencyInjection/Configuration.php

```diff
@@ -70,6 +86,32 @@ public function getConfigTreeBuilder()
             ->end()
         ;
 
+        $this->addImageSection($rootNode);
+
         return $treeBuilder;
     }
+
+    /**
+     * Configures the `contao.image` section.
+     *
+     * @param ArrayNodeDefinition $rootNode
+     */
+    private function addImageSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('image')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('bypass_cache')
```

Comment:
Did you notice the `image` array node? The configuration will look like this:

```yaml
contao:
    image:
        bypass_cache: true
        target_dir: "foo/bar"
    prepend_locale: false
```

The container parameter will be `contao.image.bypass_cache`.